internal/v2/domain/human_otp.go

```diff
@@ -27,3 +30,28 @@ const (
 func (s OTPState) Valid() bool {
 	return s >= 0 && s < otpStateCount
 }
+
+func NewOTPKey(issuer, accountName string, cryptoAlg crypto.EncryptionAlgorithm) (*otp.Key, *crypto.CryptoValue, error) {
+	key, err := totp.Generate(totp.GenerateOpts{Issuer: issuer, AccountName: accountName})
```

Comment:
unsure if we should use the otp library directly in the domain package or just have an interface for it... tests might be difficult